import.sql

```diff
@@ -0,0 +1,87 @@
+CREATE TEMP TABLE data_import (
```

Comment:
The script also runs if the database is not empty and then causes exceptions in the backend

Reply:
I am not sure what you mean. Is data_import not empty? It is a TEMP table, but it could make sense to drop it afterwards.